docs/source/concepts/LeanElasticPeaksWorkspace.rst

```diff
@@ -48,50 +147,92 @@ LeanElasticPeaksWorkspace Python Interface
 See :class:`IPeaksWorkspace <mantid.api.IPeaksWorkspace>` for
 complete API.
 
+To get peaks from an existing workspace `'name_of_peaks_workspace'`
+
 .. code-block:: python
 
     pws = mtd['name_of_peaks_workspace']
     pws.getNumberPeaks()
     p = pws.getPeak(12)
     pws.removePeak(34)
 
+To create an empty LeanElasticPeaksWorkspace then add peaks to it:
+
+.. testsetup:: createpeaks
+
+   from mantid.kernel import V3D
+
+.. testcode:: createpeaks
+
+   peaks = CreatePeaksWorkspace(NumberOfPeaks=0, OutputType="LeanElasticPeak")
+   p0 = peaks.createPeakQSample([1,1,1])
+   peaks.addPeak(p0)
+   SetUB(peaks, a=2, b=2, c=2)
+   p1 = peaks.createPeakHKL([1,2,3])
+   peaks.addPeak(p1)
+   for n in range(2):
+       print('Peak {} hkl = {h:.0f}{k:.0f}{l:.0f} q_sample = {QSample}'.format(n, **peaks.row(n)))
```

Comment:
Although smart, `**peaks.row(n))` may prove a bit too cryptic for your average user. I leave it to you to let it rest as it is or write a more explicit expression